script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt

```diff
@@ -141,17 +148,17 @@ Even a thought like that won't present itself now.
 [sha:c4b3e7b530244601b290a2609959de9773ac79db]{
 -- Page 268, Offset 32404.
 -- 中庭から屋上に向かう殺戮装置。
-A slaughtering apparatus ranging from the courtyard all the way to the rooftop.
+She is nothing more than a killing machine, making her way from the courtyard toward the rooftop.
 }
 [sha:14c073d4ed1801a1ebab0eac1553072a81aa8e3c]{
 -- Page 268, Offset 32405.
 -- たとえロア本人であろうと、今の彼女から“意識”を取り戻させる事はできない。
-Even if Roa himself were to appear, the current Ciel would not be able to regain her consciousness.
+Not even the appearance of Roa himself would be enough to shock her back to consciousness.
```

Comment:
maybe this 意識 and the other one can be like "herself"? as in "had ciel been herself" "shock herself back to herself". idk its not great but i think i like it more than "consciousness"

Reply:
How about 
```diff
-If Ciel had been conscious, it would have reminded her of a powder keg.
+If Ciel hadn't sealed off her thoughts, it would have reminded her of a powder keg.
and
-Not even the appearance of Roa himself would be enough to shock her back to consciousness.
+Not even the appearance of Roa himself would be enough to shock her out of her trance.
```

Reply:
works for me